indexer/locals.tf

```diff
@@ -124,6 +134,10 @@ locals {
           name : "BLOCKED_ADDRESSES",
           value : var.indexer_compliance_blocklist,
         },
+        {
+          name : "PG_POOL_MAX",
```

Comment:
Remove `PG_POOL_MAX` from `postgres_environment_variables` now that you've explicitly set it per service.